script/Ciel/Day 7/07_01_CIEL07_9.txt

```diff
@@ -468,44 +502,42 @@ One set before he began studying other people.
 [sha:079f1f0144cb98fc283acd65cda4c11cc3b9f00f]{
 -- Page 680, Offset 16482.
 -- 彼は精神が成熟していたが故に、童心であっても、そこにある不思議を“手の届かないもの”として処理できなかった。
-Though he still held a child-like curiosity,
- his mind was mature,
- and so he could not disregard the mysterious as merely something beyond his grasp.
+Though his mind had matured,
+ he still possessed a child-like curiosity,
+ and so he could not write off the mysterious as something beyond his grasp.
```

Comment:
Oops, yeah I think I subconsciously inverted it since this pattern (mature but with enough naïveté to pursue mystery) is a lot more common than "Childish but still mature enough to pursue the mysterious"